tests/unit/dependency/test_hdfs.py

```diff
@@ -6,6 +6,7 @@
 
 
 @pytest.mark.skipif(PY39, reason=PYARROW_NOT_AVAILABLE)
+@pytest.mark.hdfs
```

Comment:
On an individual test level, you can use this feature with `pytest.mark.{remote}` marker.